nixos/modules/services/misc/open-webui.nix

```
@@ -89,10 +89,10 @@ in
       after = [ "network.target" ];
 
       environment = {
-        STATIC_DIR = ".";
-        DATA_DIR = ".";
-        HF_HOME = ".";
-        SENTENCE_TRANSFORMERS_HOME = ".";
+        STATIC_DIR = "${cfg.statedir}/static";
+        DATA_DIR = "${cfg.statedir}/data";
+        HF_HOME = "${cfg.statedir}/hf_home";
+        SENTENCE_TRANSFORMERS_HOME = "${cfg.statedir}/transformers_home";
```

I'm getting a missing attribute for `statedir`. I'm guessing you meant to write `stateDir`?

```suggestion
        STATIC_DIR = "${cfg.stateDir}/static";
        DATA_DIR = "${cfg.stateDir}/data";
        HF_HOME = "${cfg.stateDir}/hf_home";
        SENTENCE_TRANSFORMERS_HOME = "${cfg.stateDir}/transformers_home";
```